File: src/main/java/conditionalStatements/OvenClock.java
package conditionalStatements;

import java.util.Scanner;

public class OvenClock {

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        int hour = scanner.nextInt();
        int minute = scanner.nextInt();
        int cookingMinute = scanner.nextInt();
        scanner.close();

        int totalMinute = hour * 60 + minute + cookingMinute;

        int finalHour = (totalMinute / 60) % 24;
        int finalMinute = totalMinute % 60;

        System.out.println(finalHour + " " + finalMinute);
    }

}

File: src/main/kotlin/conditionalStatements/OvenClock.kt
package conditionalStatements

import java.util.Scanner

fun main() {
    val scanner = Scanner(System.`in`)
    val hour = scanner.nextInt()
    val minute = scanner.nextInt()
    val cookingMinute = scanner.nextInt()
    scanner.close()

    val totalMinute = hour * 60 + minute + cookingMinute

    val finalHour = (totalMinute / 60) % 24
    val finalMinute = totalMinute % 60

    println("$finalHour $finalMinute")

}